### PR TITLE
add sys/eth; move sys/wifi/watchdog to sys/watchdog; make sys/eth, sys/wifi/sta PATCHable

### PR DESCRIPTION
**Pull request by chrismerck**
Ethernet endpoint and related changes for Bond Bridge Pro (to be released as `v2.16` firmware).

Render: https://bond-api-docs.s3.amazonaws.com/merck-ethernet-local.html

### CODE REVIEW

File: network/paths.yaml [hunk 1]
```diff
@@ -29,7 +36,90 @@ WifiResetPath:
...
+    and netmask, respectively. If any of these fields are present, all three must
+    be present.
+
+    For DNS, Bond always tries default DNS servers first. However, our chosen
+    DNS servers are blocked on some networks, so Bond will fall back to DNS
+    servers provided by DHCP or specified by `dns` and `dns_alt`. If `dns` or
+    `dns_alt` are provided, Bond will ignore any DNS servers proposed
+    by DHCP. This behavior was decided upon after many customer service interactions
+    which revealed an epidemic of bad ISP DNS servers which prevent what we
+    originally though was reasonable: respecting the DHCP-provided servers by default.
```
Comment (marciogranzotto) [inline network/paths.yaml:120]: maybe we should have a `dhcp` flag? Not sure if it is needed or nice to have though

File: network/paths.yaml [hunk 2]
```diff
@@ -206,19 +298,19 @@ WatchdogPatch:
...
     '401':
       $ref: ../common/responses.yaml#/Unauthorized
     '500':
       $ref: ../common/responses.yaml#/InternalServerError
   security:
   - BasicAuth: []
-  summary: Configure Wi-Fi Watchdog
+  summary: Configure Network Watchdog
   description: |
     Configure the Wi-Fi watchdog, which runs whenever the Wi-Fi station (`sta`)
```
Comment (marciogranzotto) [inline network/paths.yaml:310]:
```suggestion
    Configure the network watchdog, which runs whenever the Wi-Fi station (`sta`)
```

also, change the `which runs whenever the Wi-Fi station` to the new behavior

File: network/paths.yaml [hunk 3]
```diff
@@ -206,19 +298,19 @@ WatchdogPatch:
...
     is configured.
 
     This watchdog feature is useful on some Wi-Fi networks where the Bond
```
Comment (marciogranzotto) [inline network/paths.yaml:313]:
```suggestion
    This watchdog feature is useful on some networks where the Bond
```

**Review by marciogranzotto — APPROVED** [review]

File: network/paths.yaml (hunk 1 above)
Comment (chrismerck) [reply network/paths.yaml:120]: Well, users can already configure static IP, in which case DHCP does not run. So I think that's already covered.

**Pull request merged**